life_server/Functions/Systems/fn_cleanupRequest.sqf

```diff
@@ -7,11 +7,12 @@
     the server will then monitor when that client aborts and
     delete the weapon holders.
 */
-private ["_client","_loops"];
-_client = [_this,0,objNull,[objNull]] call BIS_fnc_param;
+params [
+    ["_client",objNull,[objNull]]
+];
 if (isNull _client) exitWith {};
 
-_loops = 0;
+private _loops = 0;
```

Comment:
this can be made into a for loop from 0 to 25 with a step of 1 (which means you don't need to specify the step. Then use the exitWith regarding a dead player as normal